app/containers/title-field.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+import { connect } from 'react-redux'
+import TitleField from '../components/title-field'
+import {
+  updateTitle,
+  activateTitleEditing,
+  updateTemporaryTitleValue,
+  deactivateTitleEditing
+} from '../actions'
+
+const makeMapStateToProps = (initialState, initialProps) => {
+  const { key } = initialProps.dat
+  const mapStateToProps = state => ({
+    dat: state.dats[key],
+    titleUnderEdit: state.titleUnderEdit
+  })
+
+  return mapStateToProps
+}
+
+const mapDispatchToProps = dispatch => ({
```

Comment:
this is hard to read, maybe let's add the `{ return ... }` around the function block?